gitlab-status/handler.go

```diff
@@ -77,6 +77,8 @@ func appendParameters(URL string, state string, desc string, context string) (st
 
 }
 
+var httpClient = http.DefaultClient
```

Comment:
No need for this pointer. Please just inline it.

Reply:
Done